ecosystem_tests/ecosystem_tests_cli/commands/blueprint_linting.py

```diff
@@ -0,0 +1,138 @@
+@ecosystem_tests.options.repo_name
+@ecosystem_tests.options.directory
+@ecosystem_tests.options.pull_request_title
+def blueprint_linting(access_token=None,
+                      repo_name=None,
+                      directory=None,
+                      pull_request_title=None):
+
+    time = str(datetime.now())
+    time = time.replace(' ', '_')
+    time = time.replace(':', '-')
+    time = time.replace('.', '-')
+
+    branch_name = time
+
+    if not access_token:
+        access_token = os.environ("GITHUB_TOKEN")
+    if not repo_name:
+        org_name = os.environ("CIRCLE_PROJECT_USERNAME")
+        repo_name = org_name + "/" + os.environ("CIRCLE_PROJECT_REPONAME")
+    if not directory:
+        directory = "/tmp/" + time
+    else:
+        directory = directory + time
+    if not pull_request_title:
+        pull_request_title = "cfy-lint autofix " + time
+
+    # Define the name of the files of interest
+    file_name = ".yaml"
+
+    command = "cfy-lint -b {} --format JSON"
+
+    # get github objects
+    g = Github(access_token)
+    # user = g.get_user()
+    git_repo = g.get_repo(repo_name)
+
+    # clone repo to dest_folder
+    repo = Repo.clone_from(git_repo.clone_url, os.path.join(
+        os.getcwd(), directory))
+
+    # check if there are issues in the blueprints
+    i = 0
+    try:
+        for root, dirs, files in os.walk(directory):
+            for file in files:
+                if file_name in file:
+                    i = i + 1
+                    # If the file matches the desired name,
+                    # execute the command on it
+                    full_path = os.path.join(root, file)
+                    full_command = command.format(full_path)
+                    p = subprocess.Popen(full_command.split(),
+                                         stdout=subprocess.PIPE,
+                                         stderr=subprocess.PIPE)
+                    stdout, stderr = p.communicate()
+                    for line in stderr.decode('utf-8').split('\r\n'):
+                        if line:
+                            raise StopIteration
+    except StopIteration:
+        command = "cfy-lint -b {} -af"
+        # create branch
+        source_branch = git_repo.default_branch
+        sb = git_repo.get_branch(source_branch)
+        git_repo.create_git_ref(
+            ref='refs/heads/' + branch_name, sha=sb.commit.sha)
+        repo.git.pull()
+        repo.git.checkout(branch_name)
+
+    if command == "cfy-lint -b {}":
+        # return 0
+        raise StopIteration
+
+    # run auto fix
+    for root, dirs, files in os.walk(directory):
+        for file in files:
+            if file_name in file:
+                # If the file matches the desired name,
+                # execute the command on it
+                full_path = os.path.join(root, file)
+                full_command = command.format(full_path)
+                print(full_path)
+                print(full_command)
```

Comment:
Use general logger:
```python
from ..logger import logger

# ...

    logger(full_path)
    logger(full_command)

```

And I prefer we add something to the stdout message that explains what we are printing, e.g.:

```python
    logger.info('Explain what we are printing: {}'.format(message))
```